src/main/java/com/conveyal/gtfs/util/GeoJsonUtil.java

```diff
@@ -0,0 +1,341 @@
+            }
+        }
+        return locationShapes;
+    }
+
+    /**
+     * Produces a {@link LocationShape} based on the values provided.
+     */
+    private static LocationShape buildLocationShape(
+        String location_id,
+        String geometry_id,
+        double geometry_pt_lat,
+        double geometry_pt_lon
+    ) {
+        LocationShape shape = new LocationShape();
+        shape.location_id = location_id;
+        shape.geometry_id = geometry_id;
+        shape.geometry_pt_lat = geometry_pt_lat;
+        shape.geometry_pt_lon = geometry_pt_lon;
+        return shape;
+    }
+
+    /**
+     * Extract the location features from file, unpack and convert to a CSV representation.
+     */
+    public static CsvReader getCsvReaderFromGeoJson(
+        String tableName,
+        ZipFile zipFile,
+        ZipEntry entry,
+        List<String> errors
+    ) {
+        FeatureCollection features = GeoJsonUtil.getLocations(zipFile, entry);
+        if (features == null || features.numFeatures() == 0) {
+            String message = "Unable to extract GeoJson features (or none are available) from " + entry.getName();
+            LOG.warn(message);
+            if (errors != null) errors.add(message);
+            return null;
+        }
+        StringBuilder csvContent = new StringBuilder();
+        if (tableName.equals(Table.LOCATIONS.name)) {
+            List<Location> locations = GeoJsonUtil.unpackLocations(features, errors);
+            csvContent.append(Location.header());
+            locations.forEach(location -> csvContent.append(location.toCsvRow()));
+        } else if (tableName.equals(Table.LOCATION_SHAPES.name)) {
+            List<LocationShape> locationShapes = GeoJsonUtil.unpackLocationShapes(features, errors);
+            csvContent.append(LocationShape.header());
+            locationShapes.forEach(locationShape -> csvContent.append(locationShape.toCsvRow()));
+        }
+        return new CsvReader(new StringReader(csvContent.toString()));
+    }
+
+    /**
+     * Convert {@link Location} and {@link LocationShape} lists to a serialized String conforming to the GeoJson
+     * standard.
+     */
+    public static String packLocations(
+        List<Location> locations,
+        List<LocationShape> locationShapes,
+        List<String> errors
+    ) {
+        FeatureCollection featureCollection = new FeatureCollection();
+        List<Feature> features = new ArrayList<>();
+        for (Location location : locations) {
+            switch (location.geometry_type) {
+                // location geometry type (polyline) rather than spec LINESTRING
+                case "polyline":
+                    LineString ls = buildLineString(getLineStings(location.location_id, locationShapes));
+                    LineString lineString = new LineString();
+                    lineString.setPoints(ls.getPoints());
+                    mil.nga.sf.geojson.Feature lineStringFeature = new mil.nga.sf.geojson.Feature();
```

Comment:
Remove package prefixes, they are already imported (multiple instances).

Reply:
Have removed where possible. In some cases the same class name is used in different packages:

`mil.nga.sf.geojson.LineString`
`mil.nga.sf.LineString`